### PR TITLE
fixed broken image of anand kumar

### LINKED ISSUE #495: Fixing the broken profile Image for "Anand Kumar Singh"
**Description**
The profile image link on the [https://meshery.io/community/](https://meshery.io/community/) and [https://meshery.io/contributors/anand-kumar-singh/](https://meshery.io/contributors/anand-kumar-singh/) is broken.

**Expected Behavior**
The profile image for Anand Kumar Singh on the above-mentioned pages should be visible.

**Screenshots**
image

bug

**Enviroment:**
 - OS: Windows 10
 - Browser: Chrome, Brave, FireFox
 - Version: [e.g. 22]
 - Device: Laptop

---
**To Reproduce**
1. Go to  [https://meshery.io/community/](https://meshery.io/community/) or [https://meshery.io/contributors/anand-kumar-singh/](https://meshery.io/contributors/anand-kumar-singh/)

**Additional Context**
The Image seems to be fine on mobile devices.

### PR DESCRIPTION
**Pull request by The-Loony-Coder**
Signed-off-by: Mritunjay <msengar121@gmail.com>

**Description**

This PR fixes #495 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

**preview**
image-fix
image-fix1

### CODE REVIEW

**Comment by hdck007** [comment]
Great work @The-Loony-Coder it looks good to me. You need to wait till one of the maintainers reviews it!! 
cc @Anita-ihuman

**Review by hdck007 — APPROVED** [review]

**Comment by The-Loony-Coder** [comment]
> Great work @The-Loony-Coder it looks good to me. You need to wait till one of the maintainers reviews it!!
> cc @Anita-ihuman

sure, thanks

**Review by leecalcote — APPROVED** [review]
Looking good, gents. Thank you, @The-Loony-Coder @hdck007 👍

**Pull request merged**
